api/utils.py

```diff
@@ -26,18 +26,25 @@ def get_collection_files_regex(limit_collection, language) -> List:
     :param limit_collection: The list of collections to limit to
     :param language: The desired language
     :return: The regular expressions to test if resource belongs to a given collection
+    if a collection is prefixed with !, we exclude it from the results!
```

Comment:
There are two types of filters: one can either ask for having a certain file or collection only (positive), which returns only the parallels with that specific file or collection or several thereof (multi-select). The other filter is the opposite: it means you want all the parallel results except the ones from a certain collection or file (negative).
This has the advantage that you can choose. Otherwise if you want to exclude collection Z, you have to give in that you want A, B,C .... Y all separately which is annoying.